pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -256,9 +257,23 @@ func validateServiceInstanceUpdate(instance *sc.ServiceInstance) field.ErrorList
 // internalValidateServiceInstanceUpdateAllowed ensures there is not a
```

Comment:
You may want to update this comment to reflect how changes by the same user are treated.

Reply:
good catch- fixed